### PR TITLE
Et3 42

### PR DESCRIPTION
**Pull request by MAliseyko**

### CODE REVIEW

File: front-end/src/Components/UserTask.jsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react/prop-types,react/destructuring-assignment,no-unused-vars,react/jsx-no-bind */
```
Comment (ala-n) [inline front-end/src/Components/UserTask.jsx]: cheater? ))
Agree that some eslint objections can be reduced in priority. Some of them even can be disabled, but try to not to do it for one file.

File: front-end/src/Main/Services/Reducers/reducers.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const initialState = {
+  userType: 'guest',
+  isAuth: false,
+  isReady: true,
+};
+
+const app = (state = initialState, action) => {
+  switch (action.type) {
+    case types.IS_LOGGED_IN: {
```
Comment (ala-n) [inline front-end/src/Main/Services/Reducers/reducers.js]: Confilicts with Julia code can appears...

File: front-end/src/Scenes/Teachers/Tasks/AddTask/Components/InputOutputSet.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable prefer-const */
```
Comment (ala-n) [inline front-end/src/Scenes/Teachers/Tasks/AddTask/Components/InputOutputSet.jsx]: Why here?

File: front-end/src/Components/UserTask.jsx (hunk 1 above)
Comment (MAliseyko) [reply front-end/src/Components/UserTask.jsx]: Yeah, i temporarily disabled these rules just to see if code works. Forgot to fix them.

File: front-end/src/Main/Services/Reducers/reducers.js (hunk 2 above)
Comment (MAliseyko) [reply front-end/src/Main/Services/Reducers/reducers.js]: This part of code is not finished yet. I was waiting for Julia's pull request to be approved, so i could finish what I wanted to do.

**Pull request merged**